DOCSP-36988 Bulk Write (#15)
mongodb/docs-pymongo: source/fundamentals/crud/bulk-write.txt

arbitrary order, which means they might run in parallel. The driver reports
any errors that occur after attempting all operations.

In the following example, the first and third operations raise an error because of the unique
constraint on ``_id``. Because the operation is unordered, only the second
and fourth operations succeed.

.. code-block:: python

   >>> requests = [
   ...     InsertOne({"_id": 1}),
   ...     DeleteOne({"_id": 2}),
   ...     InsertOne({"_id": 3}),
   ...     ReplaceOne({"_id": 4}, {"i": 1}),
   ... ]
   >>> try:
   ...     db.test.bulk_write(requests, ordered=False)
   ... except BulkWriteError as bwe:
   ...     pprint(bwe.details)
   ...
   {'nInserted': 0,
    'nMatched': 1,
    'nModified': 1,
    'nRemoved': 1,
    'nUpserted': 0,
    'upserted': [],
    'writeConcernErrors': [],
    'writeErrors': [{'code': 11000,
                     'errmsg': '...E11000...duplicate key error...',
                     'index': 0,...
                     'op': {'_id': 1}},
                    {'code': 11000,
                     'errmsg': '...',
                     'index': 2,...
                     'op': {'_id': 3}}]}

Write Concern
-------------

> {+driver-short+} runs bulk operations using the
> ``write_concern`` of the collection in which the operations are running. The

Reviewer: S: avoiding "using"

```suggestion
When {+driver-short+} runs a bulk operation, it uses the``write_concern`` of the collection in
 which the operation is running.
```
